src/components/DiagnosticsButtonRun.vue

```diff
@@ -16,7 +16,11 @@ export default Vue.extend({
   },
   computed: {
     friendlyTimeLastRun(): string {
-      return this.currentTime.to(dayjs(this.timeLastRun));
+      // Because the currentTime is updated only every second it's possible for the last-time-run to have
+      // happened after the current-time.
+      // We can't update this.currentTime because computed methods can't have side-effects, so treat an
+      // older currentTime the same as timeLastRun.
+      return this.timeLastRun.valueOf() >= this.currentTime.valueOf() ? this.currentTime.to(this.currentTime) : this.currentTime.to(dayjs(this.timeLastRun));
```

Comment:
This line is long enough that it should span multiple lines (due to linting, you'll probably need `if ()` over `?:`).

I believe you don't need explicit calls to `valueOf` for the comparison; `this.timeLastRun > this.currentTime` should be enough.  Or even `this.currentTime.to(Math.min(this.lastTimeRun, this.currentTime))`.

Reply:
I get these errors: `Operator '>' cannot be applied to types 'Date' and 'Dayjs'.`

Reply:
But this works:
```
eval("console.log(`<: ${ theDay < theDayJS }`);");
```

I don't think that's what you had in mind